### ISSUE TITLE
pAdvertising->stop(); Don't work.

### ISSUE THREAD

**Issue by m0923678421** [espressif/arduino-esp32#2853]
### Hardware:                                         ESP32-S/ESP-WROOM-32
Board:							ESP32 Dev Module
Core Installation version:			1.0.1
IDE name:						Arduino IDE
Flash Frequency:					80Mhz
PSRAM enabled:                             yes
Upload Speed:					921600
Computer OS:                                        Windows 7

### Description:
I use the BLE_iBeacon example and disable deep sleep at that sketch.
But even the code run over "pAdvertising->stop();" this line.
I can still scan this iBeacon device with smartphone.
I also try btStop(); , btStart(); and reStart(); function.
But when the code run over btStop(); again.
ESP32 will reboot itself.
Help me please.

How can I stop iBeacon and restart in sketch without reboot?

### Sketch:  (leave the backquotes for [code formatting](https://help.github.com/articles/creating-and-highlighting-code-blocks/))
```cpp

/*
   Based on Neil Kolban example for IDF: https://github.com/nkolban/esp32-snippets/blob/master/cpp_utils/tests/BLE%20Tests/SampleScan.cpp
   Ported to Arduino ESP32 by pcbreflux
*/

 
/*
   Create a BLE server that will send periodic iBeacon frames.
   The design of creating the BLE server is:
   1. Create a BLE Server
   2. Create advertising data
   3. Start advertising.
   4. wait
   5. Stop advertising.
   6. deep sleep
   
*/
#include "sys/time.h"

#include "BLEDevice.h"
#include "BLEUtils.h"
#include "BLEBeacon.h"
#include "esp_sleep.h"

#define GPIO_DEEP_SLEEP_DURATION     10  // sleep x seconds and then wake up
RTC_DATA_ATTR static time_t last;        // remember last boot in RTC Memory
RTC_DATA_ATTR static uint32_t bootcount; // remember number of boots in RTC Memory

#ifdef __cplusplus
extern "C" {
#endif

uint8_t temprature_sens_read();
//uint8_t g_phyFuns;

#ifdef __cplusplus
}
#endif

// See the following for generating UUIDs:
// https://www.uuidgenerator.net/
BLEAdvertising *pAdvertising;
struct timeval now;

#define BEACON_UUID           "8ec76ea3-6668-48da-9866-75be8bc86f4d" // UUID 1 128-Bit (may use linux tool uuidgen or random numbers via https://www.uuidgenerator.net/)

void setBeacon() {

  BLEBeacon oBeacon = BLEBeacon();
  oBeacon.setManufacturerId(0x4C00); // fake Apple 0x004C LSB (ENDIAN_CHANGE_U16!)
  oBeacon.setProximityUUID(BLEUUID(BEACON_UUID));
  oBeacon.setMajor((bootcount & 0xFFFF0000) >> 16);
  oBeacon.setMinor(bootcount&0xFFFF);
  BLEAdvertisementData oAdvertisementData = BLEAdvertisementData();
  BLEAdvertisementData oScanResponseData = BLEAdvertisementData();
  
  oAdvertisementData.setFlags(0x04); // BR_EDR_NOT_SUPPORTED 0x04
  
  std::string strServiceData = "";
  
  strServiceData += (char)26;     // Len
  strServiceData += (char)0xFF;   // Type
  strServiceData += oBeacon.getData(); 
  oAdvertisementData.addData(strServiceData);
  
  pAdvertising->setAdvertisementData(oAdvertisementData);
  pAdvertising->setScanResponseData(oScanResponseData);

}

void setup() {

    
  Serial.begin(115200);
  gettimeofday(&now, NULL);

  Serial.printf("start ESP32 %d\n",bootcount++);

  Serial.printf("deep sleep (%lds since last reset, %lds since last boot)\n",now.tv_sec,now.tv_sec-last);

  last = now.tv_sec;
  
  // Create the BLE Device
  BLEDevice::init("");

  // Create the BLE Server
  // BLEServer *pServer = BLEDevice::createServer(); // <-- no longer required to instantiate BLEServer, less flash and ram usage

  pAdvertising = BLEDevice::getAdvertising();
  
  setBeacon();
   // Start advertising
  pAdvertising->start();
  Serial.println("Advertizing started...");
  delay(100);
  pAdvertising->stop();
  /*
  Serial.printf("enter deep sleep\n");
  esp_deep_sleep(1000000LL * GPIO_DEEP_SLEEP_DURATION);
  Serial.printf("in deep sleep\n");
  */
}

void loop() {
}
```

### Debug Messages:
```
[W][esp32-hal-psram.c:30] psramInit(): PSRAM init failed!
start ESP32 0
deep sleep (0s since last reset, 0s since last boot)
[I][BLEDevice.cpp:561] getAdvertising(): create advertising
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:152] setAdvertisementData(): >> setAdvertisementData
[D][BLEAdvertising.cpp:160] setAdvertisementData(): << setAdvertisementData
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:169] setScanResponseData(): >> setScanResponseData
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 4]
[E][BLEAdvertising.cpp:174] setScanResponseData(): esp_ble_gap_config_scan_rsp_data_raw: 258 Unknown ESP_ERR error
[D][BLEAdvertising.cpp:177] setScanResponseData(): << setScanResponseData
[D][BLEAdvertising.cpp:186] start(): >> start: customAdvData: 1, customScanResponseData: 1
[D][BLEAdvertising.cpp:205] start(): - no services advertised
[D][BLEAdvertising.cpp:246] start(): << start
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 6]
[D][BLEAdvertising.cpp:256] stop(): >> stop
[D][BLEAdvertising.cpp:262] stop(): << stop
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 17]
[I][BLEAdvertising.cpp:495] handleGAPEvent(): STOP advertising
[D][BLEAdvertising.cpp:186] start(): >> start: customAdvData: 1, customScanResponseData: 1
[D][BLEAdvertising.cpp:205] start(): - no services advertised
[D][BLEAdvertising.cpp:246] start(): << start
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 6]
ets Jun  8 2016 00:22:57

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:1
load:0x3fff0018,len:4
load:0x3fff001c,len:1100
load:0x40078000,len:10088
load:0x40080400,len:6380
entry 0x400806a4
[W][esp32-hal-psram.c:30] psramInit(): PSRAM init failed!
start ESP32 0
deep sleep (0s since last reset, 0s since last boot)
[I][BLEDevice.cpp:561] getAdvertising(): create advertising
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:152] setAdvertisementData(): >> setAdvertisementData
[D][BLEAdvertising.cpp:160] setAdvertisementData(): << setAdvertisementData
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:169] setScanResponseData(): >> setScanResponseData
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 4]
[E][BLEAdvertising.cpp:174] setScanResponseData(): esp_ble_gap_config_scan_rsp_data_raw: 258 Unknown ESP_ERR error
[D][BLEAdvertising.cpp:177] setScanResponseData(): << setScanResponseData
[D][BLEAdvertising.cpp:186] start(): >> start: customAdvData: 1, customScanResponseData: 1
[D][BLEAdvertising.cpp:205] start(): - no services advertised
[D][BLEAdvertising.cpp:246] start(): << start
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 6]
[D][BLEAdvertising.cpp:256] stop(): >> stop
[D][BLEAdvertising.cpp:262] stop(): << stop
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 17]
[I][BLEAdvertising.cpp:495] handleGAPEvent(): STOP advertising
[D][BLEAdvertising.cpp:186] start(): >> start: customAdvData: 1, customScanResponseData: 1
[D][BLEAdvertising.cpp:205] start(): - no services advertised
[D][BLEAdvertising.cpp:246] start(): << start
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 6]

```

**Comment by m0923678421** [comment]
### Sketch:  (leave the backquotes for [code formatting](https://help.github.com/articles/creating-and-highlighting-code-blocks/))
```cpp
/*
   Based on Neil Kolban example for IDF: https://github.com/nkolban/esp32-snippets/blob/master/cpp_utils/tests/BLE%20Tests/SampleScan.cpp
   Ported to Arduino ESP32 by pcbreflux
*/


/*
   Create a BLE server that will send periodic iBeacon frames.
   The design of creating the BLE server is:
   1. Create a BLE Server
   2. Create advertising data
   3. Start advertising.
   4. wait
   5. Stop advertising.
   6. deep sleep

*/
#include "sys/time.h"

#include "BLEDevice.h"
#include "BLEUtils.h"
#include "BLEBeacon.h"
#include "esp_sleep.h"

#define GPIO_DEEP_SLEEP_DURATION     10  // sleep x seconds and then wake up
RTC_DATA_ATTR static time_t last;        // remember last boot in RTC Memory
RTC_DATA_ATTR static uint32_t bootcount; // remember number of boots in RTC Memory

#ifdef __cplusplus
extern "C" {
#endif

uint8_t temprature_sens_read();
//uint8_t g_phyFuns;

#ifdef __cplusplus
}
#endif

// See the following for generating UUIDs:
// https://www.uuidgenerator.net/
BLEAdvertising *pAdvertising;
struct timeval now;

#define BEACON_UUID           "8ec76ea3-6668-48da-9866-75be8bc86f4d" // UUID 1 128-Bit (may use linux tool uuidgen or random numbers via https://www.uuidgenerator.net/)

unsigned long lastTime = 0;
void setBeacon() {

  BLEBeacon oBeacon = BLEBeacon();
  oBeacon.setManufacturerId(0x4C00); // fake Apple 0x004C LSB (ENDIAN_CHANGE_U16!)
  oBeacon.setProximityUUID(BLEUUID(BEACON_UUID));
  oBeacon.setMajor((bootcount & 0xFFFF0000) >> 16);
  oBeacon.setMinor(bootcount & 0xFFFF);
  BLEAdvertisementData oAdvertisementData = BLEAdvertisementData();
  BLEAdvertisementData oScanResponseData = BLEAdvertisementData();

  oAdvertisementData.setFlags(0x04); // BR_EDR_NOT_SUPPORTED 0x04

  std::string strServiceData = "";

  strServiceData += (char)26;     // Len
  strServiceData += (char)0xFF;   // Type
  strServiceData += oBeacon.getData();
  oAdvertisementData.addData(strServiceData);

  pAdvertising->setAdvertisementData(oAdvertisementData);
  pAdvertising->setScanResponseData(oScanResponseData);

}

void setup() {


  Serial.begin(115200);
  gettimeofday(&now, NULL);

  Serial.printf("start ESP32 %d\n", bootcount++);

  Serial.printf("deep sleep (%lds since last reset, %lds since last boot)\n", now.tv_sec, now.tv_sec - last);

  last = now.tv_sec;

  // Create the BLE Device
  BLEDevice::init("");

  // Create the BLE Server
  // BLEServer *pServer = BLEDevice::createServer(); // <-- no longer required to instantiate BLEServer, less flash and ram usage

  pAdvertising = BLEDevice::getAdvertising();

  setBeacon();
  // Start advertising
  pAdvertising->start();
  Serial.println("Advertizing started...");
  delay(100);
  pAdvertising->stop();
  /*
    Serial.printf("enter deep sleep\n");
    esp_deep_sleep(1000000LL * GPIO_DEEP_SLEEP_DURATION);
    Serial.printf("in deep sleep\n");
  */
}

void loop() {
  if (millis() - lastTime > 5000) {
    if (!btStarted()) {
      btStart();
    }
    setBeacon();
    // Start advertising
    pAdvertising->start();
    Serial.println("Advertizing started...");
    delay(500);
    pAdvertising->stop();
    btStop();
    lastTime = millis();
  }else if(lastTime > millis()){
    lastTime = millis();
  }
}
```

### Debug Messages:
```
[W][esp32-hal-psram.c:30] psramInit(): PSRAM init failed!
start ESP32 0
deep sleep (0s since last reset, 0s since last boot)
[I][BLEDevice.cpp:561] getAdvertising(): create advertising
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:152] setAdvertisementData(): >> setAdvertisementData
[D][BLEAdvertising.cpp:160] setAdvertisementData(): << setAdvertisementData
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:169] setScanResponseData(): >> setScanResponseData
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 4]
[E][BLEAdvertising.cpp:174] setScanResponseData(): esp_ble_gap_config_scan_rsp_data_raw: 258 Unknown ESP_ERR error
[D][BLEAdvertising.cpp:177] setScanResponseData(): << setScanResponseData
[D][BLEAdvertising.cpp:186] start(): >> start: customAdvData: 1, customScanResponseData: 1
[D][BLEAdvertising.cpp:205] start(): - no services advertised
[D][BLEAdvertising.cpp:246] start(): << start
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 6]
[D][BLEAdvertising.cpp:256] stop(): >> stop
[D][BLEAdvertising.cpp:262] stop(): << stop
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 17]
[I][BLEAdvertising.cpp:495] handleGAPEvent(): STOP advertising
[D][BLEAdvertising.cpp:186] start(): >> start: customAdvData: 1, customScanResponseData: 1
[D][BLEAdvertising.cpp:205] start(): - no services advertised
[D][BLEAdvertising.cpp:246] start(): << start
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 6]
[D][BLEAdvertising.cpp:152] setAdvertisementData(): >> setAdvertisementData
[D][BLEAdvertising.cpp:160] setAdvertisementData(): << setAdvertisementData
[D][BLEAdvertising.cpp:169] setScanResponseData(): >> setScanResponseData
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[E][BLEAdvertising.cpp:174] setScanResponseData(): esp_ble_gap_config_scan_rsp_data_raw: 258 Unknown ESP_ERR error
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 4]
[D][BLEAdvertising.cpp:177] setScanResponseData(): << setScanResponseData
[D][BLEAdvertising.cpp:186] start(): >> start: customAdvData: 1, customScanResponseData: 1
[D][BLEAdvertising.cpp:205] start(): - no services advertised
[D][BLEAdvertising.cpp:246] start(): << start
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 6]
[D][BLEAdvertising.cpp:256] stop(): >> stop
[D][BLEAdvertising.cpp:262] stop(): << stop
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 17]
[I][BLEAdvertising.cpp:495] handleGAPEvent(): STOP advertising
[D][BLEAdvertising.cpp:186] start(): >> start: customAdvData: 1, customScanResponseData: 1
[D][BLEAdvertising.cpp:205] start(): - no services advertised
[D][BLEAdvertising.cpp:246] start(): << start
[D][BLEAdvertising.cpp:152] setAdvertisementData(): >> setAdvertisementData
[D][BLEAdvertising.cpp:160] setAdvertisementData(): << setAdvertisementData
[D][BLEAdvertising.cpp:169] setScanResponseData()S ERT_etScanR12po),eDatrw
l[Ec atEAdne 230ing.cpp:174] setScanResponseData(): esp_ble_gap_config_scan_rsp_data_raw: 258 Unknown ESP_ERR error
[D][BLEAdvertising.cpp:177] setScanResponseData(): << setScanResponseData
[D][BLEAdvertising.cpp:186] start(): >> start: customAdvData: 1, customScanResponseData: 1
[D][BLEAdvertising.cpp:205] start(): - no services advertised
[D][BLEAdvertising.cpp:246] start(): << start
Guru Meditation Error: Core  0 panic'ed (Interrupt wdt timeout on CPU0)
Core 0 register dump:
PC      : 0x40086e7e  PS      : 0x00060034  A0      : 0x8008a7f8  A1      : 0x3ffbe400  
A2      : 0x00000001  A3      : 0x00000000  A4      : 0x00000000  A5      : 0x60008054  
A6      : 0x3ffbdd48  A7      : 0x60008050  A8      : 0x80086e79  A9      : 0x3ffbe3e0  
A10     : 0x00000004  A11     : 0x00000000  A12     : 0x6000804c  A13     : 0xffffffff  
A14     : 0x00000000  A15     : 0xfffffffc  SAR     : 0x00000004  EXCCAUSE: 0x00000005  
EXCVADDR: 0x00000000  LBEG    : 0x40086db1  LEND    : 0x40086db8  LCOUNT  : 0x00000000  
Core 0 was running in ISR context:
EPC1    : 0x4000bff0  EPC2    : 0x00000000  EPC3    : 0x00000000  EPC4    : 0x40086e7e

Backtrace: 0x40086e7e:0x3ffbe400 0x4008a7f5:0x3ffbe420 0x4008b3db:0x3ffbe440 0x40081859:0x3ffbe460 0x4000bfed:0x00000000

Core 1 register dump:
PC      : 0x401aaed6  PS      : 0x00060134  A0      : 0x8014bc96  A1      : 0x3ffbc5f0  
A2      : 0x00000000  A3      : 0x80000001  A4      : 0x00000000  A5      : 0x00000001  
A6      : 0x00060a20  A7      : 0x00000000  A8      : 0x00060023  A9      : 0x3ffbc164  
A10     : 0x00000000  A11     : 0x00000001  A12     : 0x80094c5c  A13     : 0x3ffca020  
A14     : 0x00000000  A15     : 0x3ffbc2e0  SAR     : 0x00000000  EXCCAUSE: 0x00000005  
EXCVADDR: 0x00000000  LBEG    : 0x00000000  LEND    : 0x00000000  LCOUNT  : 0x00000000  

Backtrace: 0x401aaed6:0x3ffbc5f0 0x4014bc93:0x3ffbc610 0x40093931:0x3ffbc630 0x40094a49:0x3ffbc650

Rebooting...
ets Jun  8 2016 00:22:57

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:1
load:0x3fff0018,len:4
load:0x3fff001c,len:1100
load:0x40078000,len:10088
load:0x40080400,len:6380
entry 0x400806a4
[W][esp32-hal-psram.c:30] psramInit(): PSRAM init failed!
start ESP32 0
deep sleep (-140465845s since last reset, -140465845s since last boot)
[I][BLEDevice.cpp:561] getAdvertising(): create advertising
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:152] setAdvertisementData(): >> setAdvertisementData
[D][BLEAdvertising.cpp:160] setAdvertisementData(): << setAdvertisementData
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:169] setScanResponseData(): >> setScanResponseData
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 4]
[E][BLEAdvertising.cpp:174] setScanResponseData(): esp_ble_gap_config_scan_rsp_data_raw: 258 Unknown ESP_ERR error
[D][BLEAdvertising.cpp:177] setScanResponseData(): << setScanResponseData
[D][BLEAdvertising.cpp:186] start(): >> start: customAdvData: 1, customScanResponseData: 1
[D][BLEAdvertising.cpp:205] start(): - no services advertised
[D][BLEAdvertising.cpp:246] start(): << start
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 6]
[D][BLEAdvertising.cpp:256] stop(): >> stop
[D][BLEAdvertising.cpp:262] stop(): << stop
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 17]
[I][BLEAdvertising.cpp:495] handleGAPEvent(): STOP advertising
[D][BLEAdvertising.cpp:186] start(): >> start: customAdvData: 1, customScanResponseData: 1
[D][BLEAdvertising.cpp:205] start(): - no services advertised
[D][BLEAdvertising.cpp:246] start(): << start
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 6]
[D][BLEAdvertising.cpp:152] setAdvertisementData(): >> setAdvertisementData
[D][BLEAdvertising.cpp:160] setAdvertisementData(): << setAdvertisementData
[D][BLEAdvertising.cpp:169] setScanResponseData(): >> setScanResponseData
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[E][BLEAdvertising.cpp:174] setScanResponseData(): esp_ble_gap_config_scan_rsp_data_raw: 258 Unknown ESP_ERR error
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 4]
[D][BLEAdvertising.cpp:177] setScanResponseData(): << setScanResponseData
[D][BLEAdvertising.cpp:186] start(): >> start: customAdvData: 1, customScanResponseData: 1
[D][BLEAdvertising.cpp:205] start(): - no services advertised
[D][BLEAdvertising.cpp:246] start(): << start
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 6]
[D][BLEAdvertising.cpp:256] stop(): >> stop
[D][BLEAdvertising.cpp:262] stop(): << stop
[D][BLEAdvertising.cpp:152] setAdvertisementData(): >> setAdvertisementData
[D][BLEAdvertising.cpp:160] setAdvertisementData(): << setAdvertisementData
[D][BLEAdvertising.cpp:169] setScanResponseData(): >> setScanResponseData
[D][BLEDevice.cpp:563] getAdvertising(): get advertising
[E][BLEAdvertising.cpp:174] setScanResponseData(): esp_ble_gap_config_scan_rsp_data_raw: 258 Unknown ESP_ERR error
[D][BLEAdvertising.cpp:479] handleGAPEvent(): handleGAPEvent [event no: 17]
[D][BLEAdvertising.cpp:177] setScanResponseData(): << setScanResponseData
[I][BLEAdvertising.cpp:495] handleGAPEvent(): STOP advertising
[D][BLEAdvertising.cpp:186] start(): >> start: customAdvData: 1, customScanResponseData: 1
[D][BLEAdvertising.cpp:186] start(): >> start: customAdvData: 1, customScanResponseData: 1
[D][BLEAdvertising.cpp:205] start(): - no services advertised
[D][BLEAdvertising.cpp:246] start(): << start


```

**Comment by taichunmin** [comment]
I have same issue. I find a link about that. Can we fix that?
https://github.com/nkolban/esp32-snippets/issues/797

**Comment by taichunmin** [comment]
Please fix this bug.

**Comment by beegee-tokyo** [comment]
Fixed already with #3034
